<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CobotKuka</class>
 <widget class="QMainWindow" name="CobotKuka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CobotKuka</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QGridLayout" name="global_gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="connect_horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="connect_groupBox">
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="connect_ip_label">
             <property name="text">
              <string>IP : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="connect_ip_lineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>172.30.1.125</string>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::VisualMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="connect_port_label">
             <property name="text">
              <string>Port : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="connect_port_lineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string>30004</string>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::VisualMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="connect_horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="connect_status_label">
          <property name="styleSheet">
           <string notr="true">QLabel { color : red; }</string>
          </property>
          <property name="text">
           <string>Disconnected</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connect_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="send_pushButton">
        <property name="text">
         <string>-&gt; Send &lt;-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="mod_groupBox">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="mod_gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="svg_groupBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgb(246, 255, 240)</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="svg_verticalLayout">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="svg_radioButton">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>SVG</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_Mod</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="svg_actions_groupBox">
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="svg_file_label">
                   <property name="text">
                    <string>file : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="svg_file_lineEdit">
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="svg_file_pushButton">
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Open...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="text_groupBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgb(255, 240, 255)</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="text_verticalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="text_radioButton">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>TEXT</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_Mod</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="text_actions_groupBox">
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="text_font_horizontalLayout">
                   <item>
                    <widget class="QLabel" name="text_font_label">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>144</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Font : </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFontComboBox" name="text_font_fontComboBox">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="text_text_horizontalLayout">
                   <item>
                    <widget class="QLabel" name="text_text_label">
                     <property name="text">
                      <string>Your text : </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="text_text_lineEdit">
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="picture_groupBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgb(240, 255, 255)</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="picture_verticalLayout">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="picture_radioButton">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>PICTURE</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_Mod</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="picture_actions_groupBox">
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="picture_file_horizontalLayout">
                   <item>
                    <widget class="QRadioButton" name="picture_file_radioButton">
                     <property name="text">
                      <string>file : </string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">buttonGroup_Picture</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="picture_file_lineEdit">
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="picture_file_pushButton">
                     <property name="text">
                      <string>Open...</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="picture_webcam_horizontalLayout">
                   <item>
                    <widget class="QRadioButton" name="picture_webcam_radioButton">
                     <property name="text">
                      <string>webcam : </string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">buttonGroup_Picture</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="picture_webcam_pushButton">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>Open webcam</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="sketch_groupBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgb(255, 255, 240)</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="sketch_verticalLayout">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="sketch_radioButton">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>SKETCH</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_Mod</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="sketch_actions_groupBox">
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="sketch_pushButton">
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Open Sketch Window</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from the server and exit the program</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_Picture"/>
  <buttongroup name="buttonGroup_Mod"/>
 </buttongroups>
</ui>
